packages/react-drag-drop/src/next/components/DragDrop/examples/DataListDraggable.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
I would argue that this should be a demo.
cc @andrew-ronaldson

Reply:
Should we put these two directly onto the demos page, or should we create a "demos - next" tab?

Reply:
I would say demos next tab